### DISCUSSION TITLE
Inlcude creation timestamp into TimesheetEntity

### DISCUSSION THREAD

**Discussion by thomasDOTwtf** [kimai/kimai#3553] [General]
**Is your feature request related to a problem? Please describe.**
To be able to check Timesheet entries creation time, the info is needed to check timely creation of entries.

**Describe the solution you'd like**
Please extend the TimesheetEntity with adding the entry's creation time.

**Describe alternatives you've considered**
I've considered directly accessing the database.

**Comment by kevinpapst** [comment]
The "Audit logs" plugin keeps that info. And the timesheet table has a "modified at" date.
That's enough for me. If you need more, please use Doctrine events to store the info somewhere else. There is no requirement that I am aware of that needs the creation timestamp. 
But I am certain you have a use-case, do you want to share it?

**Comment by thomasDOTwtf** [comment]
Ok. I'll have a look into that plugin mentioned. Our use case is to rewards entry of timesheets if done by midnight the same day. Therefore, we'll need a possibility to see if that has been accomplished by all users.

**Comment by kevinpapst** [comment]
I'd say use the modified field. Not perfect and might allow for some edge case manipulations, but should be sufficient.
Compare if begin/end/date and modified_date are the same day.
